### PR TITLE
[Doc] Add Prerequisites to deploy broker doc

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/deployment/deploy_broker.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ We recommend you deploy ONE Broker node on each instance that hosts a BE node, a
 
 Broker nodes use the network connection to transmit data to BE nodes. When a Broker node and a BE node are deployed on the same machine, the Broker node transmit the data to the local BE node.
 
+## Before you begin
+
+Make sure you have completed the required configurations by following the instructions provided in [Deployment prerequisites](../deployment/deployment_prerequisites.md)、[Check environment configurations](../deployment/environment_configurations.md), and [Prepare deployment files](../deployment/prepare_deployment_files.md).
```
Comment (evelynzhaojie) [inline docs/deployment/deploy_broker.md:11]: punctuation

**Review by evelynzhaojie — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

org.eclipse.egit.github.core.client.RequestException: Server Error (502)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by amber-create** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Pull request merged**